lms/djangoapps/teams/static/teams/js/views/team_profile.js

```diff
@@ -50,6 +51,8 @@
                     this.discussionView.render();
 
                     this.renderTeamMembers();
+
+                    $('.sr-is-focusable').focus();
```

Comment:
Actually it should be `this.$('.page-header-main .sr-is-focusable').focus();`. There may be other elements in the view with this class.